test/views/html/MemberDetailsViewSpec.scala

```diff
@@ -35,7 +35,6 @@ class MemberDetailsViewSpec extends GmpViewSpec{
     behave like pageWithBackLink
 
     "have correct input labels" in {
-      doc must haveInputLabelWithText("nino", messages("gmp.nino") + " " + messages("gmp.nino.hint"))
```

Comment:
maybe keep the assertion just for label without the hint

Reply:
ok